### PR TITLE
chore: show next in order of priority in Dashboard

### LINKED ISSUE #1922: Update Dashboard script to show Next in order of priority

### PR DESCRIPTION
**Pull request by worstell**
fixes #1922

### CODE REVIEW

**Review by deniseli — APPROVED** [review]
niiiice

**Review by wesbillman — APPROVED** [review]

**Comment by worstell** [comment]
closing/reopening this pr to trigger the workflow script to execute for testing 😄

File: scripts/update-workflow-issue [hunk 1]
```diff
@@ -18,6 +18,21 @@ list_issues_and_prs() {
   (gh pr list -A "$user" "$@" --json number,labels,mergedAt | jq '[.[] | .["closedAt"] = .mergedAt | del(.mergedAt)]'; gh issue list -a "$user" "$@" --json number,labels,closedAt) | jq '.[]' | jq -rs 'sort_by(.closedAt) | reverse' | jq -r "$(map_to_markdown)"
 }
 
+list_prioritized_issues() {
+  gh issue list "$@" --json number,labels | jq '.[]' | jq -s '
+    map(
+      . as $issue |
```
Comment (alecthomas) [inline scripts/update-workflow-issue:24]: Wew that is some deep jq mojo @worstell 👏

Related...I'm wondering if maybe we should rewrite this script in Python at some point 😂

**Review by alecthomas — APPROVED** [review]

File: scripts/update-workflow-issue (hunk 1 above)
Comment (worstell) [reply scripts/update-workflow-issue:24]: you can thank chatgpt

Comment (alecthomas) [reply scripts/update-workflow-issue:24]: Thanks Chad!

**Pull request merged**